transform/send_aws_lambda.go

```diff
@@ -0,0 +1,171 @@
+package transform
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
+
+	"github.com/brexhq/substation/config"
+	"github.com/brexhq/substation/internal/aggregate"
+	"github.com/brexhq/substation/internal/aws"
+	"github.com/brexhq/substation/internal/aws/lambda"
+	iconfig "github.com/brexhq/substation/internal/config"
+	"github.com/brexhq/substation/internal/errors"
+	"github.com/brexhq/substation/message"
+)
+
+// Payloads greater than 256 KB in size cannot be
+// sent to an AWS Lambda function.
+const sendLambdaPayloadSizeLimit = 1024 * 1024 * 256
+
+// errSendLambdaPayloadSizeLimit is returned when data exceeds the Lambda
+// payload size limit. If this error occurs, then conditions or transforms
+// should be applied to either drop or reduce the size of the data.
+var errSendLambdaPayloadSizeLimit = fmt.Errorf("data exceeded size limit")
```

Comment:
Style: without string formatting or error wrapping we could use `[errors.New](https://pkg.go.dev/errors#New)` here

Reply:
Since every error gets wrapped in a string to identify the transform (e.g. `fmt.Errorf("transform: send_aws_lambda: %v", err)`), I'm going to leave this as-is for now for readability consistency.